analyze.py

```diff
@@ -43,15 +45,19 @@ def run(args_dict):
         args_dict['date'].append(args_dict['date'][0])
     args_dict['date'] = [datetime.strptime(x, '%m/%d/%Y') for x in args_dict['date']]
 
-    db = mdb.connect(user='{}'.format(USERNAME), host=args_dict['host'],
-                     password='{}'.format(PASSWORD), db=args_dict['db'], charset='utf8')
+    if args_dict['dbengine'] == 'mysql':
+        db = mdb.connect(user='{}'.format(USERNAME), host=args_dict['host'],
+                         password='{}'.format(PASSWORD), db=args_dict['db'], charset='utf8')
+    elif args_dict['dbengine'] == 'sqlite':
+        db = sqlite3.connect('{}.db'.format(args_dict['db']), isolation_level=None)
```

Comment:
Same thing here -- let's close this out with an `else:` statement that exits if someone tries to use an invalid DB engine.